src/main/java/org/fundacionjala/sfdc/pageobjects/opportunities/SFONewModifyPage.java

```diff
@@ -78,4 +78,8 @@ public void getSaveNewAccountButton() {
         CommonActions.clickElement(this.saveNewAccountButton);
     }
 
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)